go.mod

```diff
@@ -61,7 +61,10 @@ require (
 	sigs.k8s.io/controller-runtime v0.7.0
 )
 
-require github.com/cenkalti/backoff v2.2.1+incompatible
+require (
```

Comment:
Why not combine require parts?

Reply:
Here's the result of the `go mod tidy` auto-formatting. I'll combine it.

Reply:
Done.

Reply:
Not sure why auto format into two require parts, maybe Go version issue ;(